src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTORY = 0.75F;
+    private Node<K, V>[] table;
+    private int size;
+    private float loadFactor;
+    private float threshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        loadFactor = DEFAULT_LOAD_FACTORY;
+        threshold = table.length * loadFactor;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        putNode(key, value, table);
     }
 
     @Override
     public V getValue(K key) {
+        for (Node<K, V> node : nodes()) {
+            if (Objects.equals(key, node.key)) {
+                return node.value;
+            }
```

Comment:
you don't need to search key among all the buckets. performance will not be the best. calculate hash and search key only in list by the calculated hash

Reply:
ok